dss/index/es/schemainfo.py

```diff
@@ -0,0 +1,25 @@
+import re
+from collections import namedtuple
+from typing import Optional
+
+from dss.util.types import JSON
+
+schema_url_regex = re.compile(r'/(?P<major_version>[0-9]+)\.[0-9]+\.[0-9]+[\w\d/]*/(?P<type>[\w\d]+)(.json)?$')
+
+
+class SchemaInfo(namedtuple('SchemaInfo', ['url', 'version', 'type'])):
+    @classmethod
+    def from_json(cls, data: dict) -> Optional['SchemaInfo']:
+        if data.get('describedBy'):
+            url = data['describedBy']
+            version, doc_type = schema_url_regex.search(url).group('major_version', 'type')
+            info = cls(url, version, doc_type)
+        elif data.get('core'):
+            # TODO remove once core is not supported. Github issue #1015
+            doc_type = data['core']['type']  # type: str
+            version = data['core']['schema_version'].split(".")[0]  # type: str
```

Comment:
looking at the regexp, I think you don't need the split

Reply:
but the regex is only used in `describedBy`. `core.schema_version` is expected to have the format 1.1.1.

Reply:
You're absolutely right.